.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,5 +11,5 @@
 - [ ] Have you added an integration test for the changes?
 - [ ] Have you built the gem locally and made queries against it successfully?
 - [ ] Did you update the changelog?
-- [ ] Did you bump the package version?
+- [ ] Did you bump the package version [in the code generator](https://github.com/patch-technology/client-code-generation/blob/main/configs/ruby-config.json#L11-L12)?
```

Comment:
At some point, we might want to clarify guidelines for contributions. E.g. this repo is open-source, but outside users can't really contribute as their changes would get overwritten by the code generator. They also can't access this link.